Documents/idea document 2/ideadocument2.md

```diff
@@ -5,9 +5,38 @@
 # Data Collection
 
 # Methods
-(Thom)
+This chapter specifies which methods will be used for the neighbourhood classification. The several steps that have to be taken to extract useful information out of the data are dispayed in a pipeline diagram, after which the steps are elaborated somewhat more.
 
-# Speficications & Execution Plan
+### Overall system architecture
+The following graph shows how input data (e.g. tweets) are processed into neighbourhood classifications.
+<p align="center">
+  <img alt="Workbench electron-browser component" src="../images/pipeline.png" width="60%">
+</p>
+
+These tweets can be used for both the processing and identification of neighbourhoods and direct display on the user interface. All relevant (sport-related) tweets will be displayed.
```

Comment:
I think relevant needs to be determined by for example BM25 algorithm with certain threshold (to be set). Then it is less vague